tutor/interactive.py

```diff
@@ -148,6 +157,17 @@ def ask_questions(config: Config, run_for_prod: Optional[bool] = None) -> None:
             config,
             defaults,
         )
+    if clean_environment:
+        run_clean = click.confirm(
+            fmt.question(
+                "The -c flag you provide will delete your existing Tutor environment. "
+                "Are you sure you want to do this?"
```

Comment:
nits:
- The -c flag deletes and recreates existing Tutor environment {env_path}
- Are you sure you want to proceed with environment deletion

The sentence `Are you sure you want to do this?` looked confusing when reading it in interactive mode. It gave the impression that we want to stop the entire operation, not just cleanup.

Reply:
Well, since it's a risky function and user might remove its files, we should kinda warn them about the importance of their decision, don't we? @kdmccormick asked to change the original to this, [here](https://github.com/overhangio/tutor/pull/1086#discussion_r1681744804). @regisb do you have something about the clean-up message?

Reply:
Right, the pause for confirmation is coming directly from [Regis's original formulation of the `-c` flag](https://github.com/overhangio/tutor/issues/967#issuecomment-1903535377). So it is good to have the prompt, but I agree with @DawoudSheraz that the current version does kinda make it look like the entire operation will be cancelled if you say No:

![image](https://github.com/user-attachments/assets/02c1810b-22a1-4136-9689-1d6db95d7311)

I recommend simplifying the message to just: `Remove existing Tutor environment directory? [Y/n] `

Reply:
Sure, I changed the message.

Reply:
Thanks!